src/App.js

```diff
@@ -27,6 +28,15 @@ export default class App extends Component {
     constructor(props) {
         super(props);
 
+        firebase.initializeApp({
+            apiKey: "AIzaSyArvjqKlClk35Xsr-TsUXVtpX-ZysOGqVk",
```

Comment:
Once we have real data in the database, we probably shouldn't have the apiKey on github

Reply:
You're right. I was not thinking about the security issues when I added the key